substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/serialize/hosted/SerializationFeature.java

```diff
@@ -137,6 +140,16 @@ static void println(String str) {
         this.typeResolver = typeResolver;
     }
 
+    /**
+     * No need to deny all associated classes, only the specified class itself is registered as
+     * denied. This method is the same as
+     * {@link SerializationDenyRegistry#registerAllAssociatedClasses(ConfigurationCondition, Class)}.
```

Comment:
```suggestion
     * denied.
```
Seems like a left over from a previous version :)

Reply:
fixed